### DISCUSSION TITLE
[proof-of-concept done] Remote server support

### DISCUSSION THREAD

**Discussion by futuretim** [go-task/task#441] [Ideas]
Has remote server support been considered? I.e. running tasks remotely?

I forked and created a very simple, naive implementation of it just as a sort of proof-of-concept, does minimally what I need but would love it to be supported upstream.

The other problem is that I have no real need for anything except Linux server support so I have no ability to test otherwise.

I have remote server support at the top-level and per-task working as well as dynamic vars (although I realized the other day that the remote configuration itself does not yet have var support).

Anyway, any feedback is appreciated. Thanks!

**Comment by andreynering** [comment]
I see you already found the related issue and I just added a comment there: https://github.com/go-task/task/issues/377#issuecomment-855286264
